woocommerce/woocommerce-square
# Fix the 'Sync stock from Square' and 'Sync inventory' links on the edit product screen.

### All Submissions:



---

### Changes proposed in this Pull Request:
As reported in issue #104, the 'Sync stock from Square' and 'Sync inventory' links on the edit product screen are not working when Square is the SOR. After investigating, I discovered that this issue occurred because the event handler JavaScript code was accidentally removed by pull request https://github.com/woocommerce/woocommerce-square-private/pull/1086. This pull request restores the JavaScript code to ensure that the links work on the edit product screen.

Closes #104 

### Steps to test the changes in this Pull Request:
1. Set up the Square plugin and enable sync with inventory sync enabled.
1. Set "Square" as the SOR.
1. Navigate to edit a Square-synced product.
1. Go to the "Inventory" tab of product data and click the "Sync stock from Square" or "Sync inventory" link. (The "Sync stock from Square" link should be there for the products with inventory tracking enabled and "Sync inventory" link will be available for the product that has inventory tracking disabled)
1. Verify that both links are working as expected now

### Changelog entry
> Fix - Ensure that the 'Sync stock from Square' and 'Sync inventory' links work properly on the edit product screen.
